docs/contributor-guide/developer-guide/deploy-system/prerequisits.md

```diff
@@ -3,12 +3,256 @@
+    - `CIVIFORM_APPLICANT_AUTH_PROTOCOL="oidc"`
+    - `CIVIC_ENTITY_SHORT_NAME="<your_name>"`
+    - `CIVIC_ENTITY_FULL_NAME="City of <your_name>"`
+    - `CIVIC_ENTITY_SUPPORT_EMAIL_ADDRESS="<your_email_address>"` (Note: Email only matters if you are testing email functionality. If you put your real email address, you will receive an Email Address Verfication Request form AWS when you run the setup script.)
+    - `SENDER_EMAIL_ADDRESS="<your_email_address>"`
+    - `STAGING_PROGRAM_ADMIN_NOTIFICATION_MAILING_LIST="<your_email_address>"`
+    - `STAGING_TI_NOTIFICATION_MAILING_LIST="<your_email_address>"`
+    - `STAGING_APPLICANT_NOTIFICATION_MAILING_LIST="<your_email_address>"`
+    - `BASE_URL="https://<your_dev_account_name>.civiform.dev"` (Note: This can be anything you want it to be, but should be similar to something a civic entity might use so it can pass some validation checks in the CiviForm app. Using your dev account name works.)
+    - `STAGING_HOSTNAME="<your_dev_account_name>.civiform.dev"`
+    - `APP_PREFIX="<your_dev_account_name>"` (Note: App prefix allows you to have more than one deployment in the same account. It can be set to whatever you want.)
+    - `SSL_CERTIFICATE_ARN="<arn_of_the_SSL_cert>"` (Find the ARN by clicking into the Certificate ID on the [Certificates page](https://us-east-1.console.aws.amazon.com/acm/home?region=us-east-1#/certificates/list) in AWS.)
+### [cloud-deploy-infra](https://github.com/civiform/cloud-deploy-infra)
+1. Clone [cloud-deploy-infra](https://github.com/civiform/cloud-deploy-infra) onto your local machine.
+2. Set your local copy of `civiform-deploy` to track your local copy of `cloud-deploy-infra`.
+    - In your local copy of `civiform-deploy`, update [this line](https://github.com/civiform/civiform-deploy/blob/4624131e74ff3a417d070f0a34da55744f26c347/bin/lib/checkout.sh#L86) in the `checkout::initialize` function in `checkout.sh` to point to your local repository instead of the remote origin.
+    - Eg. change 
+    ```
+    git remote add origin http://github.com/civiform/cloud-deploy-infra
+    ```
+    to 
+    ```
+    git remote add origin <path_to_local_repo>/cloud-deploy-infra
+    ```
+
+## Setup AWS Nuke
+AWS Nuke is a package that makes it easy to remove all resources from an AWS account. You will want to run this at the end of each working session (at least at the end of each day so we don't leave resources running overnight) and anytime you need to cleanup your AWS account to test new changes. It is destructive, so it's very important to set up your config file correctly to only clear out resources from your dev account. Check out the [README for the `aws-nuke` repository](https://github.com/rebuy-de/aws-nuke) for more information about the inner workings of `aws-nuke`.
+1. Download the correct binary for your machine's architecture/OS from the [aws-nuke repo](https://github.com/rebuy-de/aws-nuke/releases). Or on mac run `brew install aws-nuke`
```

Comment:
I don't think brew is installed on all macs? You could add instructions for how to get brew or describe a different approach for the installation.

Reply:
I don't see this applied?
Would be good to at least point at how to get brew

Reply:
Applied [here](https://github.com/civiform/docs/blob/8e90bab3eb53c01501c7275612f81b060da2e3cd/docs/contributor-guide/developer-guide/deploy-system/prerequisites.md?plain=1#L168). (I renamed the file from `prerequisits` to `prerequisites` which git reads as creating a whole new file, so comments end up looking outdated.)